l10n_it_fatturapa_out/views/account_view.xml

```diff
@@ -30,7 +30,7 @@
                     attrs="{
                         'invisible': [
                             '|',
-                                ('move_type', 'not in', ('out_invoice', 'in_refund')),
+                                ('move_type', 'not in', ('out_invoice', 'out_refund')),
                                 '|',
                                     '|',
                                         ('fatturapa_attachment_out_id', '!=', False),
```

Comment:
Github non mi fa mettere il commento alla riga precisa, ma c'è un'altra occorrenza alla riga 50: come mai quella non è da sostituire?

Reply:
mi e' sfuggita, cerco di tornarci su in giornata cosi si arriva al merge

Reply:
fatto